src/cheri_insts.sail

```diff
@@ -189,6 +189,13 @@ function clause execute (CSpecialRW(cd, scr, cs1)) =
 }
 
 union clause ast = CAndPerm : (regidx, regidx, regidx)
+/*!
+ * Capability register *cd* is replaced with the contents of capability
+ * register *cs1* with the \cperms{} field set to the bitwise-AND of
+ * its previous value and bits 0 .. 10 of integer register *rs2*
+ * and the \cuperms{} field set to the bitwise and of its previous value
```

Comment:
At the very least, be consistent between `bitwise-AND` and `bitwise and`

Reply:
I hope it wasn't me who added that to the MIPS docs 🏃‍♂️